### ISSUE TITLE
Swift version?

### ISSUE THREAD

**Issue by Flitskikker** [xmartlabs/XLData#22]
This looks like a great library. Will there be a Swift version? Or does someone know a equivalent library for Swift? I'd use this one with CocoaPods, but it the XLRemoteDataStoreController requires an AFHTTPSessionManager, whereas in Swift I use Alamofire.
Otherwise I'll have to come up with sometime to do everything (data loading, progress showing, endless scrolling, filtering, empty state, error state, etc.) generically myself.

Thanks!

**Comment by mstralka** [comment]
I agree this would be awesome for Swift!  Maybe they're working to convert it like they did with Eureka for XLForm

**Comment by mtnbarreto** [comment]
Glad you find it useful! At Xmartlabs we are using another library to face same problematic, https://github.com/xmartlabs/Opera, fully written in Swift and also developed by Xmartlabs team.
The idea behind Opera is different from XLData, it uses Reactive programming and the code is more swiftly. 

Please, try it out and let me know your thoughts.
